lib/rules/indent.js

```diff
@@ -816,7 +816,6 @@ module.exports = {
 
             offsets.ignoreToken(operator);
             offsets.ignoreToken(tokensAfterOperator[0]);
-            offsets.setDesiredOffset(tokensAfterOperator[0], sourceCode.getFirstToken(node), 1);
```

Comment:
I think this line was a holdover from when the behavior of `ignoreToken` was different and tokens that weren't the first on their line could still be "ignored".